packages/core/src/__tests__/ceramic-feed.test.ts

```diff
@@ -0,0 +1,158 @@
+import { expect, describe, test, beforeEach, afterEach } from '@jest/globals'
+import { TestUtils, type StreamState, type IpfsApi } from '@ceramicnetwork/common'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+import { createIPFS, swarmConnect, withFleet } from '@ceramicnetwork/ipfs-daemon'
+import type { Ceramic } from '../ceramic.js'
+import { createCeramic as vanillaCreateCeramic } from './create-ceramic.js'
+import { Model, ModelDefinition } from '@ceramicnetwork/stream-model'
+import { take } from 'rxjs'
+import { Document } from '../feed.js'
+
+function createCeramic(
```

Comment:
do we need this special createCeramic function?  Doesn't seem like we use the extra args anywhere in this test, can we just use the createCeramic from `create-ceramic.js` directly?